.github/workflows/deploy-vercel-staging.yml

```diff
@@ -67,7 +69,10 @@ jobs:
 
           # Deploy the customer on Vercel using the customer ref
           # Store the output in a variable so we can extract metadata from it
-          VERCEL_DEPLOYMENT_OUTPUT=`CUSTOMER_REF=${CUSTOMER_REF_TO_DEPLOY} yarn deploy:customer --token $VERCEL_TOKEN`
+          echo "GIT_COMMIT_SHA" $GIT_COMMIT_SHA
+          echo "github.sha" ${{ github.sha }}
+          echo "VERCEL_GITHUB_COMMIT_SHA" $VERCEL_GITHUB_COMMIT_SHA
```

Comment:
this displays properly

Reply:
![image](https://user-images.githubusercontent.com/3807458/102398434-92ea0800-3fdf-11eb-9e6a-9b003b24a8b6.png)

https://github.com/UnlyEd/next-right-now/runs/1566025695